data/mitigation/xen/get_guest_ip.sh

```diff
@@ -23,7 +23,7 @@ expect {
 		exp_continue
         }
         \"*assword\" {
-                send \"nots3cr3t\r\"
+                send \"notXXXX3t\r\"
```

Comment:
Do you have a test module to handle this change?

Reply:
Updated files,  need verify by other team.